slimta/util/typecheck.py

```diff
@@ -0,0 +1,9 @@
+def check_argtype(val, type_, name, or_none=False):
```

Comment:
As I said [before](https://github.com/slimta/python-slimta/pull/34#discussion_r44783331) I think it's a valuable help to developers using slimta, but I will let you the final word on that anyway, so think about it and tell me your final decision :-).